app/models/search_builder.rb

```diff
@@ -8,12 +8,19 @@ class SearchBuilder < Blacklight::SearchBuilder
   include BlacklightRangeLimit::RangeLimitBuilder
   include CJKQuery
 
-  self.default_processor_chain += [:add_advanced_parse_q_to_solr, :add_advanced_search_to_solr]
+  self.default_processor_chain += [:add_edismax_advanced_parse_q_to_solr, :add_advanced_search_to_solr]
   self.default_processor_chain += [:database_prefix_search]
   self.default_processor_chain += [:modify_params_for_cjk, :modify_params_for_cjk_advanced]
   self.default_processor_chain += [:consolidate_home_page_params]
   self.default_processor_chain += [:modify_single_term_qf]
 
+  # Tweak advanced search's boolean query output to use edismax instead of dismax
```

Comment:
```suggestion
  # Tweak advanced search's boolean query output to use edismax instead of dismax.
  # By using the same (edismax) query parser for advanced search as we do for regular search,
  #  the search syntax and relevance ranking are consistent.
```